task/oci-copy-oci-ta/0.1/README.md

```diff
@@ -0,0 +1,21 @@
+# oci-copy-oci-ta task
+
+Given an `oci-copy.yaml` file in the user's source directory, the `oci-copy` task will copy content from arbitrary urls into the OCI registry.
```

Comment:
What is `oci-copy.yaml`?

Reply:
Is it this one https://github.com/ralphbean/merlinite-poc/blob/main/oci.yaml? 

Is there a description of the full schema?

Reply:
That's it @tdkchen. There is no description of the full schema at this time.

Reply:
Added in [5bc5702](https://github.com/konflux-ci/build-definitions/pull/1077/commits/5bc5702147e44cf5d7c1968216bd89efcd92c12f).